packages/mdc-chips/README.md

```diff
@@ -40,10 +40,17 @@ npm install @material/chips
 ### HTML Structure
 
 ```html
-<div class="mdc-chip-set">
-  <button class="mdc-chip">
-    <span class="mdc-chip__text">Chip content</span>
-  </button>
+<div class="mdc-chip-set" role="grid">
```

Comment:
Can we also add a note somewhere that in IE11 and Edge the keyboard navigation appears to not work, since mdc-state's focuswithin pseudoselector doesn't work.

Reply:
Sure thing

Reply:
Done